go/api/sorted_set_commands.go

```diff
@@ -192,4 +192,34 @@ type SortedSetCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/zpopmin/
 	ZPopMaxWithCount(key string, count int64) (map[Result[string]]Result[float64], error)
+
+	// Blocks the connection until it removes and returns a member with the lowest score from the
+	// first non-empty sorted set, with the given `keys`` being checked in the order they
+	// are provided.
+	// `BZPOPMIN` is the blocking variant of `ZPOPMIN``.
+	//
+	// Note:
+	//   - When in cluster mode, all `keys` must map to the same hash slot.
+	//   - `BZPOPMIN` is a client blocking command, see [Blocking Commands] for more details and best practices.
+	//
+	// see [valkey.io] for more details.
+	//
+	// Parameters:
+	//   keys - The keys of the sorted sets.
+	//   timeout - The number of seconds to wait for a blocking operation to complete. A value of
+	//     `0` will block indefinitely.
+	//
+	// Return value:
+	//   A `KeyWithMemberAndScore` struct containing the key where the member was popped out, the member
+	//   itself, and the member score. If no member could be popped and the `timeout` expired, returns `null`.
```

Comment:
```suggestion
	//   itself, and the member score. If no member could be popped and the `timeout` expired, returns `nil`.
```
or adopt to your implementation